sklearn/metrics/scorer.py

```diff
@@ -66,6 +68,52 @@ def __repr__(self):
                    "" if self._sign > 0 else ", greater_is_better=False",
                    self._factory_args(), kwargs_string))
 
+    def _get_labels_from_estimator(self, estimator, target_type):
+        """Check if estimator has a classes argument and if scorer has a
+        labels argument and return new kwargs for scorer if label not already
+        set in make score initialization"""
+        est_has_classes = hasattr(estimator, "classes_")
+
+        # if nothing is to be copied
+        if self._copy_classes is None:
+            return self._kwargs
+
+        labels_atrr_name = self._copy_classes
+
+        if labels_atrr_name != 'labels':
+            if labels_atrr_name not in signature(
```

Comment:
what do you mean when labels is ignored? 
this error is in case user passes a string which is not actually a parameter. this is mostly for user-defined functions. eg. if they have say "class_names" as the name of the parameter which is semantically equivalent to "labels", then they can pass "class_names" here and we'll copy classes from there. this check is to make sure whatever name they pass is actually a parameter in the function.